src/mindlessgen/molecules/generate_molecule.py

```diff
@@ -41,6 +41,12 @@ def generate_random_molecule(
         verbosity=verbosity,
         scale_bondlength=config_generate.scale_minimal_bondlength,
     )
+    if config_generate.contract_coords:
+        mol.xyz = contract_coordinates(
+            mol.xyz,
+            ati=mol.ati,
+            scale_bondlength=config_generate.scale_minimal_bondlength,
```

Comment:
`scale_minimal_bondlenght` is planned to be renamed/explained better in the other PR?

Reply:
Yes this should be renamend in the other PR.